code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -132,6 +132,8 @@
 	body_parts_covered = null
 	fingerprint_chance = 100
 	var/flipped = 0
+	var/use_military_format = TRUE
+	var/modified_time = 0
```

Comment:
This should not be an object-level variable - move it to where you are using it in `checktime`.
```suggestion
```